admin/src/main/resources/io/buoyant/admin/js/src/metrics_collector.js

```diff
@@ -56,8 +56,8 @@ define(['jQuery'], function($) {
     }
 
     function onAddedClients(handler) {
```

Comment:
ah! yeah expected. `addServices` and `addClients` are handlers that could be called if new clients are detected (one is for the Service-focussed dashboard and one is for the current (client) dashboard. they both listen of the 'addedClients' event, but since they belong to different dashboards they won't both be triggered).

I modified the handler in metricsCollector to include the most current metricsJson because I needed to check which services added clients belong to in order to fit them under the correct service in the Serivce dashboard. the clients dashboard doesn't care about this, so it ignores that second argument (metricsRsp) passed in on line 60 of this file. 

Maybe `onAddedClients(addServices)` is confusingly named - the addServices method is really "check if any of the added clients are under a service we haven't seen before, and if they are, add that service" - perhaps I should name it something other than addServices?